ci-operator/step-registry/storage/conf/wait-for-csi-migration/storage-conf-wait-for-csi-migration-commands.sh

```diff
@@ -94,5 +94,59 @@ function wait_for_nodes() {
     done
 }
 
+function nodes_stable() {
+    # Check that the node is Ready
+    oc wait --for=condition=Ready=True node --all --timeout=0 || return 1
+    # Check the node is schedulable
+    if oc get node -o yaml | grep "unschedulable"; then
+        return 1
+    fi
+}
+
+function cluster_stable() {
+    oc wait --all --for=condition=Progressing=False clusteroperators.config.openshift.io --timeout=0 || return 1
+    oc wait --all --for=condition=Available=true clusteroperators.config.openshift.io --timeout=0 || return 1
+    oc wait --all --for=condition=Degraded=false clusteroperators.config.openshift.io --timeout=0 || return 1
```

Comment:
Just to make sure, are these `--for` checks case insensitive? Asking because there's  `false` and `False`

Reply:
Good question, they're `string`, so using True/False would be better. Fixed.